internal/kafka/command_consumer_group_list.go

```diff
@@ -7,12 +7,12 @@ import (
 	"github.com/confluentinc/cli/v3/pkg/output"
 )
 
-func (c *consumerGroupCommand) newListCommand() *cobra.Command {
+func (c *consumerCommand) newGroupListCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List Kafka consumer groups.",
 		Args:  cobra.NoArgs,
-		RunE:  c.list,
+		RunE:  c.listGroups,
```

Comment:
Follow the format of our other commands!
```suggestion
		RunE:  c.groupList,
```